searchcore/src/tests/proton/matching/request_context/request_context_test.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include <vespa/eval/eval/tensor_spec.h>
+#include <vespa/eval/tensor/serialization/typed_binary_format.h>
+#include <vespa/eval/tensor/tensor.h>
+#include <vespa/eval/tensor/test/test_utils.h>
```

Comment:
I think we should try to deprecate this one in favor of the eval/eval/test library and using tensor engine more directly